modules/ROOT/pages/dashboard-custom-config.adoc

```diff
@@ -81,27 +83,27 @@ image::dashboard-custom-config-rows.png[Dashboard Row Configurations]
 |===
 2+| Row Configuration Options for Custom Dashboards
 | Title | Title for your row.
-| Show Title | For making the title visible to end users.
-| Up Arrow icon | For moving a row up in the dashboard when there is more than one row in the dashboard.
-| Down Arrow icon | For moving a row down when there is more than one row in the dashboard.
-| Trashcan icon | For deleting the row _and any charts_ it contains from the dashboard.
+| Show Title | Make the title visible to end users.
+| Up Arrow icon | Move a row up in the dashboard when there is more than one row in the dashboard.
+| Down Arrow icon | Move a row down when there is more than one row in the dashboard.
+| Trashcan icon | Delete the row and any charts it contains from the dashboard.
 |===
 
 [[dashboard_metadata]]
 === View Metrics on a Custom Dashboard
 
-You can use the Metadata tab for high-level custom dashboard configurations to view metrics on a custom dashboard.
+Use the *Metadata* tab for high-level custom dashboard configurations to view metrics on a custom dashboard.
 
 [[dashboard_config]]
 image::dashboard-custom-metadata.png[Dashboard Metadata]
 
 [%header,cols="1,4"]
 |===
 2+| Metadata Available for Custom Dashboards
-| Last Updated At | Date and time of last update to the dashboard.
-| Last Updated By | Username associated with the Last Updated At value.
-| Created At | Date and time the dashboard was created.
-| Created By | Username of person who created the dashboard.
+| Last Updated at | Date and time of last update to the dashboard.
+| Last Updated by | Username associated with the Last Updated At value.
+| Created at | Date and time the dashboard was created.
+| Created by | Username of person who created the dashboard.
 | Current Version | Version number of the dashboard. The number increments each time you save modifications to the dashboard.
```

Comment:
Let's say "The version increments" or "The version number increments" as I had to think a moment what number we meant.